python/src/aiconfig/editor/server/server.py

```diff
@@ -102,6 +104,10 @@ def load_model_parser_module() -> FlaskResponse:
 def load() -> FlaskResponse:
     state = get_server_state(app)
     request_json = request.get_json()
+    if not request_json.keys() <= {"path"}:
```

Comment:
cool trick, is this the best/standard way to validate requests in flask? 

Just a note that FastApi handles this by making request data pydantic objects

Reply:
This is unrelated to Flask. I'm just taking the dict keys as a set and comparing them to the set containing "path".

Reply:
Can we keep it simple and say `if len(request_json.keys()) != 0 or request_json.keys() != {"path"}`? using `__lt__` is not intuitive to read or understand. I get that this logic is "elegant", but I had to play around to figure it out
<img width="141" alt="Screenshot 2023-12-28 at 14 04 28" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/2fa10c5e-b937-41ea-989f-58e68314fcaa">

Reply:
Sure, added here https://docs.google.com/document/d/1YwUlIQZX8CiuvtbVR4OvpFQ2nphFITRD_XZR3CDNtoE/edit#bookmark=id.8h0jqqaolx9z